### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by bkostanetskyi**
[DEMO LINK](https://bkostanetskyi.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://bkostanetskyi.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,11 +4,41 @@
...
+          <li class="item"><a class="link" href="#">GADGETS</a></li>
+          <li class="item"><a class="link" href="#">TABLETS</a></li>
+          <li class="item"><a class="link" href="#">PHOTO</a></li>
+          <li class="item" ><a class="link" href="#">VIDEO</a></li>
+        </ul>
+      </nav>
+
+    </header>
+
+
```
Comment (Andriy-Shymkiv) [inline src/index.html:41]: please pay attention to formatting your code. look at [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.logo {
+  max-width: 40px;
+  max-height: 40px;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:19]: no need to set max width here. logo is already 40px

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+
+.list {
```
Comment (Andriy-Shymkiv) [inline src/style.css:21]: reset margin for your list

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+  box-sizing: border-box;
+}
+
+.item:last-child {
+  margin-right: 0;
+}
+
+.item:first-child {
+  margin-left: 0;
+}
```
Comment (Andriy-Shymkiv) [inline src/style.css:48]: you can just:
```css
.item:not(:last-child) {
  margin-right: 20px;
}
```

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
better to use prefix `nav__` for your header items to keep clear BEM methodology here

File: src/index.html [hunk 5]
```diff
@@ -4,11 +4,42 @@
...
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+
```
Comment (Andriy-Shymkiv) [inline src/index.html:42]: [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) rule number 5 and 13. pay attention to your code style!

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,64 @@
...
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 22.5px 0 ;
+
+}
+
+.nav {
+  margin: 0;
```
Comment (Andriy-Shymkiv) [inline src/style.css:24]: no need to reset margin for nav, it is zero by default

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,64 @@
...
+.is-active {
+  color: #00acdc;
+
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
```
Comment (Andriy-Shymkiv) [inline src/style.css:58]: you repeating yourself at least three times. so better to move this to some variable

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,64 @@
...
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 22.5px 0 ;
```
Comment (Andriy-Shymkiv) [inline src/style.css:19]: not a good approach. better to center your items with flexbox

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
1. pay attention to your tests, because they are failing
2. request a review only after all comments fixes from mentors.
3. if you need some help, please ask it in your group chat or  QnA session

File: src/index.html (hunk 1 above)
Comment (Andriy-Shymkiv) [reply src/index.html]: not fixed!

File: src/style.css (hunk 8 above)
Comment (Andriy-Shymkiv) [inline src/style.css:19]: set margin to zero here:
```css
margin: 0;
```

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
please apply all changes

File: src/index.html [hunk 9]
```diff
@@ -4,11 +4,40 @@
...
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        alt="Moyo"
+        class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
```
Comment (xiimiik) [inline src/index.html:25]: lets improve code style

```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#" class="nav__link">Samsung</a>
          </li>
```

File: src/index.html [hunk 10]
```diff
@@ -4,11 +4,40 @@
...
+          <li class="nav__item"><a href="#" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a href="#"
+            class="nav__link "
+           data-qa="hover">Laptops & Computers</a>
```
Comment (xiimiik) [inline src/index.html:30]:
```suggestion
            <a 
              href="#"
              class="nav__link "
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,67 @@
...
+  box-sizing: border-box;
+}
+
+.nav__item :hover {
+  color: var(--bgcolor);
+}
+
+.is-active {
+  color: var(--bgcolor);
+
```
Comment (xiimiik) [inline src/style.css:54]:
```suggestion
```

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,67 @@
...
+  justify-content: center;
+  align-items: center;
+  height: var(--height-header);
+  color: #000;
+  text-decoration: none;
+  position: relative;
+  box-sizing: border-box;
+}
+
+.nav__item :hover {
```
Comment (xiimiik) [inline src/style.css:48]:
```suggestion
.nav__item:hover {
```

**Pull request closed without merging**